models/tfidf_model/tfidf.py

```diff
@@ -0,0 +1,102 @@
+import math
+from collections import Counter
+import re
+import numpy as np
+from tfidf_model.config import *
+from tfidf_model.utils import *
+print(PATH)
+
+class tfidf_sum():
+    def __init__(self):
+        self.path = PATH
```

Comment:
Probably better to have `__init__(self, path): ...` , but given that this is just a temporary code to be changed to something that doesn't read the files, probably no need to care about that.